internal/config/config.go

```diff
@@ -418,6 +421,16 @@ func peerFromCR(p metallbv1beta2.BGPPeer, passwordSecrets map[string]corev1.Secr
 		return nil, fmt.Errorf("invalid keepaliveTime %q", p.Spec.KeepaliveTime)
 	}
 
+	// by frr docs connect time should be between 1 - 65535 (max value for unsigned integer)
+	if 1 > p.Spec.ConnectTime.Duration.Seconds() || p.Spec.ConnectTime.Duration.Seconds() > math.MaxUint16 {
```

Comment:
This can be added as a validation annotation in the crd. The reason why we have other fields validated here is because originally the configuration was coming from a configmap

Reply:
Sure! did it